### PR TITLE
refs #21350 - dispatch router ssl/tls configuration

### PR DESCRIPTION
**Pull request by stbenjam**
Needs https://github.com/theforeman/puppet-qpid/pull/88

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -114,6 +118,8 @@
   Integer[0, 65535] $qpid_router_broker_port = $foreman_proxy_content::params::qpid_router_broker_port,
   String $qpid_router_logging_level = $foreman_proxy_content::params::qpid_router_logging_level,
   Stdlib::Absolutepath $qpid_router_logging_path = $foreman_proxy_content::params::qpid_router_logging_path,
+  Optional[String] $qpid_router_ssl_ciphers = $foreman_proxy_content::params::qpid_router_ssl_ciphers,
+  Optional[Array] $qpid_router_ssl_protocols = $foreman_proxy_content::params::qpid_router_ssl_protocols,
```
Comment (ekohl) [inline manifests/init.pp]: I think this could even be ```Optional[Array[String]]```

Comment (stbenjam) [reply manifests/init.pp]: Updated

**Comment by stbenjam** [comment]
I released puppet-qpid 4.2.0 and bumped metadata.json here.  Can you look again @ekohl? Thanks!

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**